src/pages/ProfilePage.js

```diff
@@ -62,13 +62,17 @@ const propTypes = {
         partnerUserID: PropTypes.string,
     }),
 
+    /** Indicated whether the report data is loading */
+    isLoadingReportData: PropTypes.bool,
+
     ...withLocalizePropTypes,
 };
 
 const defaultProps = {
     // When opening someone else's profile (via deep link) before login, this is empty
     personalDetails: {},
     loginList: {},
+    isLoadingReportData: true,
```

Comment:
Where does this prop come from?

Reply:
What's the weird error we see in this scenario? Out of curiosity.